mod/caching/set/type_plus_right/metric/wikirate_company.rb

```diff
@@ -1,24 +1,10 @@
 # cache # of companies with values for left metric
 include Card::CachedCount
 
-ensure_set do
-  Type::MetricValue
-end
+ensure_set { Type::MetricValue }
 
-expired_cached_count_cards :set=>Type::MetricValue, :on=>[:create, :delete] do |changed_card|
-  # update company count with values
-  result = [
-    # cache number of the metric values related to this metric value set
-    # for some query
-    changed_card.metric_card.fetch(trait: :wikirate_company)
-  ]
-  # it is to refresh the related metric value to the company
-  # refer to ltype_rtype/metric/wikirate_company.rb
-  if (metric_card = changed_card.metric_card) &&
-     metric_card.type_id == MetricID &&
-     (company_card = changed_card.company_card) &&
-     company_card.type_id == WikirateCompanyID
-    result.push(Card.fetch("#{metric_card.name}+#{company_card.name}"))
-  end
```

Comment:
Frankly, I couldn't make sense of this so I deleted it.  the result isn't even a search card, so this appears to be a hack